samples/subsys/fs/fat_fs/boards/sam4e_xpro.overlay

```diff
@@ -0,0 +1,6 @@
+&hsmci {
```

Comment:
Not sure why there isn't a compliance issue reported, but there should be a SPDX license header.